data/gromet/examples/README.md

```diff
@@ -1,8 +1,17 @@
-## GroMEt examples for testing
+## File Types in Example Directories
 
 The following files appear in the simple GroMet examples (not all have been generated for every example).
 
 The following are likely of most interest:
 
 - `*.py` : The source code file.
+- `*_gromet_FN_manual_diagram.pdf` : A manual drawing of the gromet wiring diagram
```

Comment:
Can you update this file to remove mentions of files that are not actually in the repo?

Reply:
Yeah I updated this to remove a couple lines of files that don't exist anymore and added a couple lines of additional files I added to deal with some bugs in the gromets in v0.1.4 that I believe are now wrapped into the v0.1.5 update.